asab/storage/elasticsearch.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, app, service_name, config_section_name='asab:storage'):
 		super().__init__(app, service_name)
 		self.Loop = app.Loop
 
-		self.ESURL = asab.Config.get(config_section_name, 'elasticsearch_url')
+		self.URL = asab.Config.get(config_section_name, 'elasticsearch_url')
```

Comment:
@mithunbharadwaj All methods here need to be properly tested. Also, the self.ESURL change will affect some of our deployments, so please make sure you notify everyone in Slack before merging.